verification/curator-service/ui/cypress/support/commands.ts

```diff
@@ -47,14 +47,14 @@ export function addCase(
     });
 }
 
-export function login(): void {
+export function login(name?: string, email?: string, roles?: string[]): void {
```

Comment:
please use an option struct instead of position arguments of the same type to avoid mistakes
opts {
  name;
  email,
  roles
}

Reply:
Done